### PR TITLE
Add Working SQL Injection Module

### PR DESCRIPTION
**Pull request by reggieag**
This adds a working SQL injection module and closes out https://github.com/reggieag/NYU_PCS_project/issues/27

see fuzzer/modules/sql_injection/README.md for more information on the strategies created

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ services:
         - "api_postgres_db"
       
     auth_service:
+      command: "/src/clients.yml"
```
Comment (reggieag) [inline docker-compose.yml:29]: This passes in the argument "/src/clients.yml" to the docker container. 🤷‍♂️ not sure why it wasn't working for me before

File: fuzzer/modules/sql_injection/Compile/StdOut.txt [hunk 2]
```diff
@@ -1,87 +0,0 @@
-[04/18/2021 00:10:59] Generating grammar...
```
Comment (reggieag) [inline fuzzer/modules/sql_injection/Compile/StdOut.txt:1]: We are no longer using RESTler so i pulled out the Compile directory.

File: fuzzer/modules/sql_injection/api.py [hunk 3]
```diff
@@ -1,126 +0,0 @@
-import os
```
Comment (reggieag) [inline fuzzer/modules/sql_injection/api.py:1]: This was the RESTler based logic that is being deprecated.

File: fuzzer/modules/sql_injection/strategies/lib/auth_request.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
+from oauthlib.oauth2 import BackendApplicationClient
```
Comment (reggieag) [inline fuzzer/modules/sql_injection/strategies/lib/auth_request.py:1]: yeah... this should be DRYed up. I might do that later, but will focus on the presentation instead for now.

Comment (reggieag) [reply fuzzer/modules/sql_injection/strategies/lib/auth_request.py]: DRYed it

File: fuzzer/modules/api/setup.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+from setuptools import setup, find_packages
```
Comment (reggieag) [inline fuzzer/modules/api/setup.py:1]: Making the shared API code a library and installing in the docker container

**Review by zhengqixi — APPROVED** [review]

**Pull request merged**